data/assets/examples/timeframe/simple.asset

```diff
@@ -0,0 +1,37 @@
+-- This example creates a union out of two simpler TimeFrameIntervals and uses it for a
+-- SceneGraphNode. The first TimeFrameInterval covers January 1st, 2000 and the second
+-- TimeFrameInterval covers March 1st, 2002. The resulting TimeFrameUnion will cover both
+-- January 1st, 2000 and March 1st, 2002.
+
+local Node = {
+  Identifier = "TimeFrameUnion_Example_Simple",
+  TimeFrame = {
+    Type = "TimeFrameUnion",
+    TimeFrames = {
+      -- The first TimeFrameInterval for the first day
+      {
+        Type = "TimeFrameInterval",
+        Start = "2000 JAN 01 00:00:00.000",
+        End = "2000 JAN 01 23:59:59.999"
+      },
+      -- The second TimeFrameInterval for the second day
+      {
+        Type = "TimeFrameInterval",
+        Start = "2002 MAR 01 00:00:00.000",
+        End = "2002 MAR 01 23:59:59.999"
+      }
+    }
+  },
+  GUI = {
+    Name = "TimeFrameUnion Example (Simple)",
+    Path = "/Examples"
```

Comment:
This comment goes for all the other PRs as well, but I would really like to have a description in the UI as well.  Just found myself dragging in the asset and then thinking - "Hm, what was this example supposed to show now again?". Without the description, there is no way of finding that out in the UI